static/app/components/profiling/ProfilingOnboarding/profilingOnboardingModal.tsx

```diff
@@ -212,16 +225,73 @@ function SelectProjectStep({
   );
 }
 
-function AndroidInstallSteps() {
+interface ProjectSdkUpdateProps {
+  minSdkVersion: string;
+  project: Project;
+  sdkUpdates: RequestState<ProjectSdkUpdates | null>;
+}
+function ProjectSdkUpdate(props: ProjectSdkUpdateProps) {
+  if (props.sdkUpdates.type !== 'resolved') {
+    return <div>{t('Verifying Sentry SDK')}</div>;
+  }
+
   return (
     <Fragment>
-      <li>
-        <StepTitle>{t('Update your projects SDK version')}</StepTitle>
-        <p>
+      <SDKUpdatesContainer>
+        <SdkUpdatesPlatformIcon platform={props.project.platform ?? 'unknown'} />
+        <span>{props.project.name}</span>
+      </SDKUpdatesContainer>
+      {props.sdkUpdates.data === null ? (
+        <SdkUpdatesText>
+          {t('This project is using the latest SDK version.')}
+        </SdkUpdatesText>
+      ) : (
+        <SdkUpdatesText>
           {t(
-            'Make sure your SDKs are upgraded to at least version 6.0.0 (sentry-android).'
+            'For profiling to function, we requires you to update your Sentry SDK to version %s or higher.',
+            props.minSdkVersion
           )}
-        </p>
+        </SdkUpdatesText>
+      )}
+    </Fragment>
+  );
+}
+
+const SdkUpdatesText = styled('p')`
+  margin-top: ${space(1.5)};
+  padding-left: ${space(4)};
+`;
+
+// doesnt use space because it's just off by 2px all the time
+const SdkUpdatesPlatformIcon = styled(PlatformIcon)`
+  margin-right: 10px;
+`;
+
+const SDKUpdatesContainer = styled('div')`
+  display: flex;
+  align-items: center;
+  margin-top: ${space(1.5)};
+  font-size: ${p => p.theme.fontSizeLarge};
+`;
+
+function AndroidInstallSteps({
+  project,
+  organization,
+}: {
+  organization: Organization;
+  project: Project;
+}) {
+  const sdkUpdates = useProjectSdkUpdates({organization, projectId: project.id});
```

Comment:
This fires a new request for each project select right? Should `useProjectSdkUpdates` only take the organization as a prop and let the per project check be done by the onboarding component?

Reply:
@Zylphrex I can just make it load the data once and then just return resolve it